packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -657,6 +655,7 @@ export class MgtFileList extends MgtTemplatedComponent {
             tabindex="0"
             class="file-item"
             @keydown="${this.onFileListKeyDown}"
+            @keyup="${this.onFileListKeyUp}"
```

Comment:
Why use @keyup when we could use @focus? 
Focus is more expressive of why we want to fire the event, and will fire if this element receives focus due to some other mechanism than then user pressing tab